frontend_lib/src/css/Variable.styl

```
@@ -44,6 +44,9 @@ metadataFontSize = 12px
 popupWidth = 420px
 bigIconSize = 20px
 
+oneAuthorAvatarSize = 30px // INFO - GB - 2021-08-26 - 30px is one author avatar's size
+twoAuthorsAvatarSize = 20px // INFO - GB - 2021-08-26 - 20px is two authors avatar's size
```

The comment lines are now too much I think (the variable names are enough).

```suggestion
oneAuthorAvatarSize = 30px
twoAuthorsAvatarSize = 20px
```